### PR TITLE
Add rollup to build setup

### PR DESCRIPTION
**Pull request by noxan**

### CODE REVIEW

**Comment by lcsrinaldi** [comment]
@noxan, for the build step not to break you have to add `- rollup.config.js` to the Circle CI configuration.

**Comment by lcsrinaldi** [comment]
Cool, it seems much simpler than webpack. Can you go through some points that would make it better for us? Webpack seems more mature, even though I saw that facebook uses Rollup for React.

File: package.json [hunk 1]
```diff
@@ -18,18 +18,17 @@
...
+  "browser": "dist/index.umd.js",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/picter/prisma.git"
   },
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "yarn build:lib && yarn build:umd && yarn build:umd:min",
-    "build:watch": "webpack -w",
-    "build:lib": "babel src --out-dir lib --ignore *.spec.js,*.story.js,__test-helpers__,__story-helpers__",
```
Comment (lcsrinaldi) [inline package.json]: Why remove this one? Rollup has an option to export de library without bundling?

Comment (noxan) [reply package.json]: Sorry, mainly wanted to use the regular build script alias.

**Comment by noxan** [comment]
Rollup has the benefit to me of being much simpler and lightweight. We still utilise the power webpack for our apps, but keep the library as lightweight as possible. But there are still some details to be discussed.

**Comment by noxan** [comment]
Maybe we can use `babel` alone for now and later maybe replace `webpack` with `rollup`.

**Comment by noxan** [comment]
I'll close it for now, maybe we need it another day, then we can come back.

**Pull request closed without merging**